fuzzylite/CMakeLists.txt

```diff
@@ -229,6 +250,12 @@ if(FL_BUILD_TESTS)
 #    set_tests_properties(ListTags PROPERTIES PASS_REGULAR_EXPRESSION "[0-9]+ tag")
 endif()
 
+if(FL_BUILD_SHARED OR FL_BUILD_STATIC)
```

Comment:
Can you explain why PRIVATE?

Reply:
As the executable target `fl-bin` cannot be linked to, the include directories for the target are set as private as they are only needed during the build process.